pkg/controller/snapshot_create_test.go

```diff
@@ -27,10 +27,11 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-var timeNow = time.Now().UnixNano()
+var timeNow = time.Now()
+var timeNowStamp = timeNow.UnixNano()
```

Comment:
Is it necessary to convert this to UnixNano?  Can we use timeNow directly in the tests?

Reply:
No, in volumesnapshot, the type of  `CreationTime` in VolumeSnapshotStatus is `*metav1.Time`, while in `CSIVolumeSnapshotSource` , the type for`CreationTime` is `*int64`, so keep two types of time is convenient for us to construct `snapshot` and `content`.